pkgs/development/libraries/science/astronomy/indilib/default.nix

```
@@ -52,7 +56,7 @@ stdenv.mkDerivation rec {
     description = "Implementation of the INDI protocol for POSIX operating systems";
     changelog = "https://github.com/indilib/indi/releases/tag/v${version}";
     license = licenses.lgpl2Plus;
-    maintainers = with maintainers; [ hjones2199 ];
-    platforms = platforms.linux;
+    maintainers = with maintainers; [ hjones2199 snick ];
+    platforms = platforms.all;
```

```suggestion
    platforms = platforms.unix;
```

also systemd is not supported in darwin so this likely should stay like it is